test/framework/src/main/java/org/elasticsearch/test/InternalTestCluster.java

```diff
@@ -1718,13 +1682,6 @@ public void fullRestart() throws Exception {
         fullRestart(EMPTY_CALLBACK);
     }
```

Comment:
It'd be neat to look at usages of `fullRestart` and see if we've implemented all of them in our full cluster restart qa tests.

Reply:
Like, later. Not as part of this.